README

```diff
@@ -36,6 +36,8 @@ More detailes are [here](http://icedtea.classpath.org/wiki/HeapStats/information
 * Linux x64 / x86_64 / AArch32
 * Oracle JDK / OpenJDK 6u18 or later
 
+**NOTE: We recommend not to use on JDK 8u262 or later due to JFR backport**
```

Comment:
```suggestion
**NOTE: For JDK 8u262 or later, we recommend to use JDK Flight Recorder instead of HeapStats since we focus to contribute OpenJDK.**
```

Reply:
Hmm... we cannot commit to contribute to JFR, and I cannot to say we are focusing JFR.

Reply:
I see.

```suggestion
**NOTE: For JDK 8u262 or later, we recommend to use built-in agent, JDK Flight Recorder, instead of HeapStats**
```